wordle-web/src/App.vue

```diff
@@ -2,11 +2,38 @@
   <header>
     <div class="wrapper">
       <nav>
-        <RouterLink to="/">Home</RouterLink> | <RouterLink to="/wordle">Wordle</RouterLink> |
-        <RouterLink to="/about">About</RouterLink>
+        <v-layout>
+          <v-app-bar>
+            <v-app-bar-nav-icon icon="mdi-air-horn" to="/wordle"></v-app-bar-nav-icon>
+            <RouterLink to="/wordle">
```

Comment:
instead of using routerLink you could add a prop to the title such as ` @click="$router.push('/')`